### DISCUSSION TITLE
Offline route navigation on ocean are for boat users

### DISCUSSION THREAD

**Discussion by vidhyablrgithub** [osmandapp/OsmAnd#18320] [Q&A]
Team,

I need to update my Android application to support offline navigation on ocean area. To achieve this, I am thinking of using OsmAnd SDK/Library as part of my application. Can you please advise, if this SDK is available for free for nautical view and navigation?

**Comment by christianCasRo** [comment]
Already answered in [#72 ](https://github.com/osmandapp/osmand-api-demo/issues/72)
If you have you own app from routes you can use AIDL API an then when select
 your rute you need Osmand to navigate. 
If you use the full library you need to ask (I supose from personal use is kinda free but if you are doing an app to sell it ...)
